Source/SwiftLintBuiltInRules/Rules/Lint/ClassDelegateProtocolRule.swift

```diff
@@ -107,7 +108,7 @@ private extension ProtocolDeclSyntax {
 private extension TypeSyntax {
     func isObjectOrDelegate() -> Bool {
         if let typeName = `as`(IdentifierTypeSyntax.self)?.typeName {
-            return typeName == "AnyObject" || typeName == "NSObjectProtocol" || typeName.hasSuffix("Delegate")
+            return (typeName == "AnyObject" || typeName == "NSObjectProtocol" || typeName.hasSuffix("Delegate")) && typeName != "Actor"
```

Comment:
`typeName != "Actor"` is always true if the condition before the `&&` is true. I think you may want to add `|| typeName == "Actor"`.